data/completions/bash/kata-runtime

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#---------------------------------------------------------------------
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Thanks for adding the SPDX license header (as this will keep https://github.com/kata-containers/tests/blob/master/.ci/static-checks.sh#L196 happy with this PR :)

I'd actually remove the license comment lines above it as that SPDX line and a copyright are sufficient. For example, you could make it the same as we have in the `Makefile`:

- https://github.com/kata-containers/runtime/blob/master/Makefile#L1..L4

Reply:
@jodh-intel @grahamwhaley thx. Removed the extra lines in the headers and just kept the copyright and the SPDX line.